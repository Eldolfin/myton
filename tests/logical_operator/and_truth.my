# False and nil are false.
print(False and "bad")  # expect: False
print(None and "bad")  # expect: None

# Everything else is true.
print(True and "ok")  # expect: ok
print(0 and "ok")  # expect: 0
print("" and "ok")  # expect: ""
