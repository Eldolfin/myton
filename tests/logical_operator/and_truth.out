False
None
ok
0

